src/main/resources/templates/error.html

```diff
@@ -12,7 +20,9 @@
 }
   </style>
 </head>
+
+<div th:replace="fragments/topnav :: navbar"> </div>
```

Comment:
The error page was very empty and offered no way back. I added the nav bar and styling to it to fix this dead end.